### PR TITLE
Maths-Magicians: Components

### PR DESCRIPTION
**Pull request by theodoku**
1. Create a new directory called components.
2. Inside components, create a new Calculator.js and Calculator.css
3. In Calculator.js, create a React component that matches design
4. Create a button folder with Button.js and Button.css

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  CALCULATOROPERATORS,
+  NUM_1,
+  NUM_2,
+  NUM_3,
+  NUM_4,
+} from '../utilities/consts';
+import Button from '../button/Button';
+import './Calculator.css';
+
+export default class Calculator extends Component {
```
Comment (TedLivist) [inline src/components/Calculator.js]: - [x] This component should be a functional component, right now it is a class component. Kindly fix that.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @theodoku,

Good job so far!
✔️ Use of child component in the calculator
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- [ ] Kindly remove these boilerplate files as they are not needed

  mjd

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 1 above)
Comment (theodoku) [reply src/components/Calculator.js]: Calculator component changed to a functional component👍

**Review by levy002 — APPROVED** [review]
## Hello @theodoku 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Status: APPROVED :fire:  :fire: 

## Highlights
- Professional README :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**